src/engine/Join.cpp

```diff
@@ -843,3 +823,29 @@ void Join::hashJoin(const IdTable& dynA, size_t jc1, const IdTable& dynB,
              << ", size = " << dynRes->size() << "\n";
 }
 
+// ___________________________________________________________________________ 
+template <int A_WIDTH, int B_WIDTH, int TABLE_WIDTH>
+void Join::addCombinedRowToIdTable(
+  const std::decay_t<typename IdTableStatic<A_WIDTH>::const_row_type>& rowA,
+  const std::decay_t<typename IdTableStatic<B_WIDTH>::const_row_type>& rowB,
```

Comment:
I will create a path for the `IdTable` s.t. this type can become a little less ugly.

Reply:
How is that going? (I'm currently trying to avoid the headache of updating the branch via master, until it has a distinct benefit.)

Reply:
We will (probably tomorrow) merge an important update to the IdTable. Maybe at the beginning of next week we can update your branch together to the new master. 
I don't want it to be your problem, that we worked at the same mechanisms (`IdTable` and `CallFixedSize` at the same time.

Reply:
Thank you for your help with merging! (See [17fe1a0](https://github.com/ad-freiburg/qlever/pull/821/commits/17fe1a067619ee94251670769c12c27119616662) for the merge.)